pom.xml

```diff
@@ -17,7 +17,7 @@
     </properties>
 
     <dependencies>
-  
+
```

Comment:
Sorry a bit nitpicky here, is it possible to remove the new added line ? Just for the sake of keeping the file's commit history clear.

Reply:
Yes for sure, missed it while cleaning pre-committing